uaclient/config.py

```diff
@@ -585,7 +585,7 @@ def warn_about_invalid_keys(self):
             LOG.warning('legacy "ua_config" found in uaclient.conf')
             LOG.warning("Please do the following:")
             LOG.warning(
-                "  1. run `pro config set field=value` for each"
+                "  1. run `sudo pro config set field=value` for each"
```

Comment:
It seems we also have that same suggestion on `postinst`. I think we should update it there as well

Reply:
Should be good now @lucasmoura, thanks!